src/lib/utils/emojiAddressAvatar.ts

```diff
@@ -0,0 +1,82 @@
+const colors = [
```

Comment:
minor: you cannot know this yet but we agreed at some point to do all file names in `kebap case`, so should be emoji-address-avatar.ts

Reply:
ah yes I mean I could have noticed